tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
+    "noFallthroughCasesInSwitch": true,
```

Comment:
This might cause issues elsewhere because I recall seeing a fall through being used somewhere. Could be wrong though...It certainly might catch someone off guard? Dunno...

Reply:
afiact a violation of `noFallthroughCasesInSwitch` will result in build breaking and not a silent error.